index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Divyansh Singh</title>
+   <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <div>
+    <header>
+    <nav class="header">
+       <ul class="list">
+           <li class="item" id="name"><a href="#">Divyansh Singh</a></li>
+        </ul>
+        <img id="header-image" src="resources/Union.png" alt="menu">
+    </nav>
+
+    </header>
+    <section>
+        <h1 class="heading" id="main">Divyansh Singh</h1>
+    <h2 class="heading" id="job">Software Developer</h2>
+        <p class="para">
+            I can help you build a product , feature or website.
+             Look through some of my work and experience!If you 
+             like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+            
+        </p>
+       
+    </section>
+    <img src="resources/Devider.png" class="divider" alt="divider">
```

Comment:
- [ ] It will be better to remove this image and add this code for `para` class to make a divider 
```
padding-bottom: 20px;
border-bottom: 2px solid #646369;
margin: 0 20px;
```